internal/client/blueprints.go

```diff
@@ -10,158 +10,158 @@ import (
 
 // PostTOMLBlueprintV0 sends a TOML blueprint string to the API
 // and returns an APIResponse
-func PostTOMLBlueprintV0(socket, blueprint string) *APIResponse {
-	body, err := PostTOML(socket, "/api/v0/blueprints/new", blueprint)
-	if err != nil {
-		return err
+func PostTOMLBlueprintV0(socket, blueprint string) (*APIResponse, error) {
+	body, resp, err := PostTOML(socket, "/api/v0/blueprints/new", blueprint)
+	if resp != nil || err != nil {
```

Comment:
I believe the distinction between client errors and API errors are not usually made in Golang. Would it be possible to treat them all as `error` return values and switch over their types from the caller if we want to make the distinction instead?

Reply:
Well, that's kinda how I had it originally but @ondrejbudai wanted separate errors so I changed it. I had an API Error that implemented the error interface, but has the extra functions to get multiple errors from it, and was converting client errors to a new error type id and message.